src/chapters/azure-intro/studio.rst

```diff
@@ -1,134 +1,8 @@
-
-.. todo:: linux subsystem and dotnet / az CLI installations page
-
-.. note::
-
-   Before continuing make sure you have the tools you need. If you do not have the linux subsystem set up with the dotnet CLI tool refer to the ref::`cli-setup` page.
-
-Publish the API
----------------
-
-There are multiple ways to publish your project and even more to customize how it is packaged. In this course we will focus on the `self-contained<https://docs.microsoft.com/en-us/dotnet/core/deploying/#publish-self-contained>`_ strategy with options to build a single executable project file. Rather than using our IDE we will use the ``dotnet`` CLI tool to get comfortable working from the command line.
-
-First navigate to your cloned repo (solution) directory. In the cloning example above the path to that directory was ``~/coding-projects/azure/coding-events-api``:
-
-.. sourcecode:: bash
-
-   $ cd ~/coding-projects/azure/coding-events-api
-
-From within the solution directory run the following command to publish your first Release:
-
-.. sourcecode:: bash
-   :caption: make sure to run this from the root (solution) directory
-
-   $ dotnet publish -c Release
-
-This will publish to ``CodingEventsAPI/bin/Release/netcoreapp3.1/linux-x64/publish/``
-
-Notice that it automatically published as a self-contained, single (executable) file, built to execute on the ``linux-x64`` runtime. These defaults were set in the ``CodingEventsAPI/CodingEventsAPI.csproj`` configuration file by the following attributes:
-
-.. sourcecode:: xml
-   :caption: CodingEventsAPI/CodingEventsAPI.csproj
-
-   <?xml version="1.0" encoding="utf-8"?>
-   <Project Sdk="Microsoft.NET.Sdk.Web">
-      <PropertyGroup>
-         <SelfContained>true</SelfContained>
-         <PublishSingleFile>true</PublishSingleFile>
-         <RuntimeIdentifier>linux-x64</RuntimeIdentifier>
-
-These defaults are the equivalent of running the publish command with the following options:
-
-.. sourcecode:: bash
-   :caption: make sure to run this from the root (solution) directory
-
-   $ dotnet publish -c Release -r linux-x64 -p:PublishSingleFile=true 
-
-
-.. tip::
-
-   If you change the ``-r`` option to a different `RID value <https://docs.microsoft.com/en-us/dotnet/core/rid-catalog>`_ you can build for other runtimes as needed while still using the defaults for the other options.
-
-Execute the API
----------------
-
-Within the ``CodingEventsAPI/bin/Release/netcoreapp3.1/linux-x64/publish/`` directory is the single executable file ``CodingEventsAPI``. All you need to do to run it is execute that file. 
-
-.. sourcecode:: bash
-
-   $ ./CodingEventsAPI/bin/Release/netcoreapp3.1/linux-x64/publish/CodingEventsAPI
-
-Now you can navigate to `https://localhost:5001`_ and view the Swagger API documentation!
-
-Running on a Remote Machine
-===========================
+===================================
+Studio: Running on a Remote Machine
+===================================
 
```

Comment:
its occurred to me that nowhere in the lessons have we described this (relative) terminology of "remote vs local". its innate to us now but i think very new to students. 

think its worth describing how these are relative terms that describe the relationship between "where you code / develop" and "where you host outside (remote)"

Reply:
Whereas I agree terms should be defined especially if students are hazy, I actually think it's explained in this write-up.